src/components/InputTodo.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+
+const InputTodo = (props) => {
+  const { addTodoProps } = props;
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (inputText.title.trim()) {
+      addTodoProps(inputText.title);
+      setInputText({
+        title: '',
+      });
+    }
+  };
+
+  return (
+    <form onSubmit={handleSubmit} className="form-container">
+      <input
+        type="text"
+        placeholder="Add Todo..."
+        value={inputText.title}
+        name="title"
+        className="input-text"
+        onChange={onChange}
+        required
+      />
+      <button type="button" className="input-submit">
+        <FaPlusCircle
+          style={{
+            color: 'darkcyan',
+            fontSize: '20px',
+            marginTop: '2px',
+          }}
```

Comment:
-When the` FaPlusCircle` is clicked, an event is not fired to add a todo. 
I think implementing a function to this button will make your project nicer and complet
<img width="738" alt="Screen Shot 2022-02-15 at 8 21 33 AM" src="https://user-images.githubusercontent.com/83285992/154021582-5deeea37-7a48-48e1-8f54-dbe9b80b651e.png">

Reply:
Hi @mrparkersson
Thank you very much for the feedback. I will address each and every one of them.
thanks again.